CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,9 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 ## Unreleased
 
+### Fixed
+- Allow s3 style CoreWeave uris for reference artifacts. (@estellazx in https://github.com/wandb/wandb/pull/9979)
```

Comment:
This section is not in the proper place. Also, is this really a fix? It looks more like an addition to me.

Reply:
Looks like it happened due to a bad merge: https://github.com/wandb/wandb/pull/9979/commits/c7fd18b409ab8497346246b8bd43145c98bfd72a

Reply:
good to know. where is it suppose to be? after Notable Changes?